### ISSUE TITLE
Where is the adapter file used for trimming?

### ISSUE THREAD

**Issue by liyawang** [terraref/computing-pipeline#220]
### Description

When trimming reads with following command:

java –jar Trimmomatic-0.35/trimmomatic-0.35.jar PE -phred33 -trimlog trimlogPE.txt SampleA_R1.fastq.gz SampleA_R2.fastq.gz SampleA_R1.PE.fastq.gz SampleA_R1.SE.fastq.gz SampleA_R2.PE.fastq.gz SampleA_R2.SE.fastq.gz ILLUMINACLIP:adapters.fa:2:30:10 SLIDINGWINDOW:5:20 LEADING:20 TRAILING:20 MINLEN:60 2> trim.out

An adapters.fa file is needed but I couldn't find it. Could you let me know where to find it? Thanks.

Also how do I interpret the following file name?
IDTW_PCRfree_PI365512_ATTACTCG-TATAGCCT_Sorghum_I549_L1_R1.fastq.gz_PE.fq.gz

Sample name is PI365512, what is Library name and Read Group ID?

**Comment by phloz** [comment]
I have attached the adapter.fa file.  It was basically created based on the attached pdf file from Illumina.

The file names were created by Hudson Alpha, who sequenced this data.  In this case, IDTW_PCRfree_PI365512_ATTACTCG-TATAGCCT_Sorghum_I549_L1_R1.fastq.gz_PE.fq.gz:

As you stated, PI365512, is the Sample name. 
ATTACTCG-TATAGCCT is the barcode.
PCRfree is significant meta data about the library(GATK --pcr_indel_model flag is specifically for PCRfree reads) 
I've used IDTW as Hudson Alpha's unique identifier for the sample.

In regard to Read Group: 
Since all of the paired-end sequence files are each from a single accession most of the read group info isn't necessary.  Using the Sample name or Sample name and the unique identifier will suffice used as Sample and the Read Group ID within this pipeline.

[adapters.fa.zip](https://github.com/terraref/computing-pipeline/files/661597/adapters.fa.zip)
[illumina-adapter-sequences.pdf](https://github.com/terraref/computing-pipeline/files/661599/illumina-adapter-sequences.pdf)
